src/app/contexts/EventTrackingContext/index.test.tsx

```diff
@@ -31,23 +45,34 @@ type Props = {
 // eslint-disable-next-line react/prop-types
 const Wrapper = ({
   children,
-  pageData = fixtureData,
+  atiData,
+  pageData,
+  isNextJs = false,
   pageType = STORY_PAGE,
+  pathname = '/pidgin/tori-51745682',
+  service = 'pidgin',
   toggles = defaultToggles,
 }: PropsWithChildren<Props>) => (
   <RequestContextProvider
     bbcOrigin="https://www.test.bbc.com"
     pageType={pageType}
     isAmp={false}
     isApp={false}
-    service="pidgin"
-    pathname="/pidgin/tori-51745682"
+    isNextJs={isNextJs}
+    service={service}
+    pathname={pathname}
   >
-    <ServiceContextProvider service="pidgin">
+    <ServiceContextProvider service={service}>
       <ToggleContextProvider toggles={toggles}>
-        <EventTrackingContextProvider pageData={pageData}>
-          {children}
-        </EventTrackingContextProvider>
+        {atiData ? (
+          <EventTrackingContextProvider atiData={atiData}>
+            {children}
+          </EventTrackingContextProvider>
+        ) : (
+          <EventTrackingContextProvider pageData={pageData}>
+            {children}
+          </EventTrackingContextProvider>
+        )}
```

Comment:
Is this necessary? Can we just pass both in? 
```suggestion
          <EventTrackingContextProvider pageData={pageData} atiData={atiData}>
            {children}
          </EventTrackingContextProvider>
```

Reply:
Passing both in now but required a couple of small changes to source code and tests